extensions/wikia/WikiaMobile/js/ads_run.js

```diff
@@ -1,30 +1,21 @@
 /*global require*/
 require(
 	[
-		'jquery', 'JSMessages', 'wikia.window', 'wikia.document', 'wikia.log', require.optional('wikia.abTest'),
-		'ext.wikia.adEngine.adEngine', 'ext.wikia.adEngine.adConfigMobile',
-		'ext.wikia.adEngine.messageListener'
+		'jquery',
+		'JSMessages',
+		'ext.wikia.adEngine.adEngine',
+		'ext.wikia.adEngine.adConfigMobile',
+		'ext.wikia.adEngine.messageListener',
+		'wikia.window',
+		'wikia.document',
+		'wikia.krux',
+		'wikia.log',
+		require.optional('wikia.abTest'),
 	],
-	function ($, msg, win, doc, log, abTest, adEngine, adConfigMobile, messageListener) {
+	function ($, msg, adEngine, adConfigMobile, messageListener, win, doc, krux, log, abTest) {
 		'use strict';
 
-		var minZerothSectionLength = 700,
-			minPageLength = 2000,
-			mobileTopLeaderBoard = 'MOBILE_TOP_LEADERBOARD',
-			mobileInContent = 'MOBILE_IN_CONTENT',
-			mobilePreFooter = 'MOBILE_PREFOOTER',
-			mobileTaboola = 'NATIVE_TABOOLA',
-			logGroup = 'ads_run',
-			logLevel = log.levels.info,
-			$firstSection = $('h2[id]').first(),
-			$footer = $('#wkMainCntFtr'),
-			firstSectionTop = ($firstSection.length && $firstSection.offset().top) || 0,
-			infoboxSelectors = ['table[class*=infobox], div[class*=infobox], div[id*=infobox]'],
-		// TODO: clean up wgAdDriverUseAdsAfterInfobox
-			infoboxAdEnabled = win.wgAdDriverUseAdsAfterInfobox && abTest && abTest.inGroup('WIKIAMOBILE_ADS_AFTER_INFOBOX', 'YES'),
```

Comment:
A little clean up is possible: WIKIAMOBILE_ADS_AFTER_INFOBOX is no longer a running experiment.